pkg/reconciler/v1alpha1/testing/functional.go

```diff
@@ -122,6 +133,45 @@ func WithServiceLabel(key, value string) ServiceOption {
 	}
 }
 
+// WithResourceRequirements attaches resource requirements to the service
+func WithResourceRequirements(resourceRequirements corev1.ResourceRequirements) ServiceOption {
+	return func(svc *v1alpha1.Service) {
+		svc.Spec.RunLatest.Configuration.RevisionTemplate.Spec.Container.Resources = resourceRequirements
+	}
+}
+
+// AddVolume adds a volume to the service
+func AddVolum(name, mountPath string, volumeSource corev1.VolumeSource) ServiceOption {
```

Comment:
`AddVolum`**e**

Also: Should this be called `WithAddedVolume` or something along these lines to comply with the rest of these?

Reply:
Good catch!   Also, renamed to func to WithVolume.   Is the new name fine?